terraform/modules/data-share-service/cloudfront.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_cloudfront_distribution" "gdx_data_share_poc" {
+  provider = aws.us-east-1
+
+  origin {
+    domain_name = aws_lb.load_balancer.dns_name
+    origin_id   = "${var.environment}-gdx-data-share-poc-lb"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "http-only"
+      # Required but irrelevant - we do not use HTTPS to talk to LB
+      origin_ssl_protocols = ["TLSv1.2"]
+      # AWS enforce a maximum of 60s, but we can request more if desired.
+      # See https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesOriginResponseTimeout
+      origin_read_timeout = 60
+    }
+  }
+
+  enabled         = true
+  is_ipv6_enabled = false
+
+  #  logging_config {
```

Comment:
Do we have a ticket/something tracking this logging set up?

Reply:
Added logging now